projects/breach-notice-and-delius/src/main/resources/application.yml

```diff
@@ -14,6 +15,10 @@ spring:
         query.mutation_strategy.global_temporary:
             create_tables: false
             drop_tables: false
+        connection:
+          autocommit: false
+    open-in-view: false
```

Comment:
Do we still need these?  Again not a problem, just wondering if they were only needed for testing